marklogic-data-hub-central/ui-custom/src/components/Section/Section.tsx

```diff
@@ -64,6 +65,7 @@ const Section: React.FC<Props> = (props) => {
     <div className="section" style={divStyle}>
       <header style={headerStyle}>
         <span>{props.title}</span>
+        {props.title === "Recently Visited" && <RecentClear title="Clear"></RecentClear>}
         {props.collapsible && <div className="collapse-container">
```

Comment:
When the Clear button and Expand/Collapse buttons both exist in the Section, it looks like this:
<img width="824" alt="Screen Shot 2022-04-04 at 9 55 03 AM" src="https://user-images.githubusercontent.com/477757/161597579-f110d60f-949a-4c4d-b3ef-9ffcae569e0e.png">

The two buttons should align to the right, with space between.

You can align them to the right by:

- Setting the Section header to display: flex
- Setting the title span to flex-grow: 1
- Setting the clear-button and collapse-container spans to flex-grow: 0

Then also add the following to clear-button and collapse-container: margin-left: 10px

Reply:
Almost!
<img width="819" alt="Screen Shot 2022-04-04 at 9 26 22 PM" src="https://user-images.githubusercontent.com/477757/161678919-b862d6bd-eaf7-46f4-803d-0cbeadf4d856.png">
The pair of collapse buttons should not be split. Apply the margin-left to collapse-container, NOT collapse-button.